blood.go

```diff
@@ -0,0 +1,16 @@
+package faker
+
+// Gender is a faker struct for Gender
+type Blood struct {
+	Faker *Faker
+}
+
+// Name returns a Gender name for Gender
+func (f Blood) Name() string {
+	return f.Faker.RandomStringElement([]string{"A+", "A-", "B+", "B-", "AB+", "AB-", "O+", "O-"})
+}
+
+// Abbr returns a Gender name for Gender
+func (f Blood) Abbr() string {
+	return f.Faker.RandomStringElement([]string{"A+", "A-", "B+", "B-", "AB+", "AB-", "O+", "O-"})
```

Comment:
Please move this list to a constant on the top like in

https://github.com/jaswdr/faker/blob/e683f8deff33c49fd58480f19006c62a984c1a9b/beer.go#L6

this way you can reuse it in your tests.